### ISSUE TITLE
Any info on marketplace transactions?

### ISSUE THREAD

**Issue by Afrokillaa** [ilmoi/awesome-solana-nfts#28]
Have you found any resources on how to interact with the popular NFT marketplaces? I'm mainly interested in Magiceden transactions, how to buy/sell NFTs via a script. I have searched a lot online, but can't seem to find good resources.

**Comment by ilmoi** [comment]
they have their API open now, so I'd follow examples there - https://api.magiceden.dev/

**Comment by Afrokillaa** [comment]
Yeah, I saw that, they only support getting the data and views listings/sales, etc. But I can't seem to find any good resource online how to actually make the Magiceden Solana transactions on the blockchain, either with Rust or Solana js. I find only examples of how to mint NFTs, but no examples on how to buy/interact with the Magiceden smart contracts. I can see the program IDs, but as I'm very new to Web3 I can't seem to figure out how to structure the Solana transaction correctly.  

The only thing I found was this: https://github.com/BL0CK-X/blockchain-api . But this library basically takes in your HTTP requests and then they do the Solana transactions on their end. It's also not scalable as the free credits are limited.
